code/datums/emotes.dm

```diff
@@ -121,11 +121,12 @@
 		//SKYRAT EDIT CHANGE END
 
 	var/user_turf = get_turf(user)
-	for(var/mob/ghost in GLOB.dead_mob_list)
-		if(!ghost.client || isnewplayer(ghost))
-			continue
-		if(ghost.stat == DEAD && ghost.client && user.client && (ghost.client.prefs.chat_toggles & CHAT_GHOSTSIGHT) && !(ghost in viewers(user_turf, null)))
-			ghost.show_message("<span class='emote'>[FOLLOW_LINK(ghost, user)] [dchatmsg]</span>")
+	if(user.client)
+		for(var/mob/ghost in GLOB.dead_mob_list)
```

Comment:
```suggestion
		for(var/mob/ghost as anything in GLOB.dead_mob_list)
```

Reply:
but why
that's just editing more code from tg to be non-modular
what would that even do